playbooks/streisand.yml

```diff
@@ -31,7 +31,8 @@
     - shadowsocks
     - ssh
     - tinyproxy
-    - tor-bridge
+    - role: tor-bridge
+      when: not streisand_ci
```

Comment:
I would maybe add a brief comment here about why we don't run this role under CI similar to what's in `tests/README.md`.